code/datums/status_effects/debuffs.dm

```diff
@@ -654,14 +654,15 @@
 	var/mob/living/carbon/dreamer = owner
 
 	if(dreamer.mind?.has_antag_datum(/datum/antagonist/vampire))
-		if(istype(dreamer.loc, /obj/structure/closet/coffin))
-			dreamer.adjustBruteLoss(-1, FALSE)
-			dreamer.adjustFireLoss(-1, FALSE)
-			dreamer.adjustToxLoss(-1)
-			dreamer.adjustOxyLoss(-1)
-			dreamer.adjustCloneLoss(-0.5)
-			if(dreamer.HasDisease(/datum/disease/critical/heart_failure) && prob(25))
-				for(var/datum/disease/critical/heart_failure/HF in dreamer.viruses)
+		var/mob/living/carbon/human/V = owner
```

Comment:
I dislike the single-letter-variable here but it's small in scope, so good for now.
Please use longer variable names in the future

Reply:
will do.